src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -1977,6 +1982,12 @@ graphicsPath = do
   getResourcePath >>= setResourcePath . (<> ps)
   return mempty
 
+svgSourceExtn :: Text -> Text
+svgSourceExtn src = if T.isSuffixOf (T.pack ".svg") src then
```

Comment:
The idiomatic way to format `if`...`then`...`else` in Haskell is to put each of these key words on their own line, together with the corresponding expression:

```haskell
if test
then result1
else result2
```